source/NVDAObjects/UIA/__init__.py

```diff
@@ -861,11 +861,14 @@ def findOverlayClasses(self,clsList):
 		# Support Windows Console's UIA interface
 		if (
 			self.windowClassName == "ConsoleWindowClass"
-			and self.UIAElement.cachedAutomationId == "Text Area"
 			and config.conf['UIA']['winConsoleImplementation'] == "UIA"
 		):
-			from .winConsoleUIA import winConsoleUIA

```

Comment:
Could you add a `findExtraOverlayClasses` method to the winConsoleUIA module? See NVDAObjects.IAccessible.mozilla.